sdk/python/lib/pulumi/provider/server.py

```diff
@@ -276,17 +278,16 @@ async def _call_response(self, result: CallResult) -> proto.CallResponse:
             urns = await asyncio.gather(*(r.urn.future() for r in resources))
             deps[k] = proto.CallResponse.ReturnDependencies(urns=urns)
 
-        # Since `return` is a keyword, we need to pass the args to `CallResponse` using a dictionary.
-        resp = {
-            "return": ret,
-            "returnDependencies": deps,
-        }
+        failures = []
         if result.failures:
-            resp["failures"] = [
+            failures = [
                 proto.CheckFailure(property=f.property, reason=f.reason)
                 for f in result.failures
             ]
-        return proto.CallResponse(**resp)
+        resp = proto.CallResponse(returnDependencies=deps, failures=failures)
+        # Since `return` is a keyword, we need to use setattr: https://developers.google.com/protocol-buffers/docs/reference/python-generated#keyword-conflicts
+        setattr(resp, "return", ret)
```

Comment:
This was the only way to write this to keep mypy happy about the constructor arguments

Reply:
But seems runtime doesn't like it... We'll need to rejig this somehow to keep both happy...